keyboards/ungodly/launch_pad/rules.mk

```
@@ -0,0 +1,32 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
```

```suggestion
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
```
Having lite enabled is recommended.